tests/setup.sh

```diff
@@ -24,6 +24,7 @@ OPENSC_VERSION=$(opensc-tool -i | grep OpenSC | sed -e "s/OpenSC 0\.\([0-9]*\).*
 if [[ "$OPENSC_VERSION" -le "25" ]]; then
     SUPPORT_ED448=0
 fi
+SUPPORT_ED448=0
```

Comment:
some testing artifact that should not made it to the final PR?

Reply:
good catch